src/index.sass

```diff
@@ -45,8 +46,24 @@ body,html
 	height: 100%
 body
 	margin: 0
-	background-color: #333
 	position: relative
+	//background-color: #333

```

Comment:
minor: remove commented code

Reply:
I let this in because it's the background used to make the screenshots.